releasenotes/template.yaml

```diff
@@ -0,0 +1,36 @@
+# This YAML file describes the format for specifying a release notes entry for Istio.
+# This should be filled in for all user facing changes.
+
+# Kind describes the type of change that this represents. 
+# Valid Values are:
+# - BugFix -- Describes a bug fixed
+# - SecurityFix -- Used to specify that this change represents a security fix. 
+# - Feature -- Used to specify a new feature that has been added. 
+# - Test -- Used to describe additional testing added. This file is optional for
+# tests, but included for completeness.
+Kind: BugFix
```

Comment:
nit: Kind -> kind seems more standard

Reply:
Are you thinking lowercase for everything? I'd like to be consistent.

Reply:
Referencing Kubernetes' YAML files, I see that all fields are lowercase. Updating here to match. Thanks John.